### PR TITLE
*: add update dependency script

### PR DESCRIPTION
**Pull request by coocood**

### CODE REVIEW

**Comment by tiancaiamao** [comment]
what if the local git repository is not the same as Godep.json says?  @coocood 
`github.com/pingcap/kvproto/pkg/... ` seems not just check kvproto, it update other packages too.

Then I run `godep restore` and get this:

    ➜  tidb git:(master) ✗ godep restore
    # cd /Users/genius/project/src/github.com/ngaut/tso/client; git checkout 3e62d377d3edbc20520c7934964615f8e91e27cd
    fatal: reference is not a tree: 3e62d377d3edbc20520c7934964615f8e91e27cd
    godep: error restoring dep (github.com/ngaut/tso/client): exit status 128
    # cd /Users/genius/project/src/github.com/ngaut/tso/proto; git checkout 3e62d377d3edbc20520c7934964615f8e91e27cd
    fatal: reference is not a tree: 3e62d377d3edbc20520c7934964615f8e91e27cd
    godep: error restoring dep (github.com/ngaut/tso/proto): exit status 128
    # cd /Users/genius/project/src/github.com/ngaut/tso/util; git checkout 3e62d377d3edbc20520c7934964615f8e91e27cd
    fatal: reference is not a tree: 3e62d377d3edbc20520c7934964615f8e91e27cd
    godep: error restoring dep (github.com/ngaut/tso/util): exit status 128
    godep: Error restoring some deps. Aborting check.

@ngaut

**Comment by coocood** [comment]
@tiancaiamao Do you mean you want to keep your local git repository in an old version?
Why you want to run `godep restore`?

File: updatedep.sh [hunk 1]
```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+mv _vendor/* .
```
Comment (shenli) [inline updatedep.sh]: Add usage or comment.

**Comment by tiancaiamao** [comment]
LGTM
rebase master please~

**Comment by coocood** [comment]
@shenli PTAL

**Comment by shenli** [comment]
LGTM

**Pull request merged**